pkg/boot/align/align_test.go

```diff
@@ -1,42 +1,62 @@
 // Copyright 2015-2022 the u-root Authors. All rights reserved
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
-
 package align
 
 import (
 	"testing"
 )
 
 func TestAlignUpPageSize(t *testing.T) {
-	pageMask = 0x03 // 4 - 1.
 	for _, tt := range []struct {
-		name string
-		val  uint
-		want uint
+		name      string
+		val       uint
+		alignSize uint
+		want      uint
 	}{
 		{
-			name: "below",
-			val:  uint(0x02),
-			want: uint(0x04),
+			name:      "below",
+			val:       uint(0x02),
+			alignSize: uint(0x04),
+			want:      uint(0x04),
```

Comment:
This test should probably check that the returned alignment is correct.

Reply:
Using `want` as the expected return. Does that make sense to you ?